main.go

```diff
@@ -0,0 +1,136 @@
+// +build go1.15
+
+package main
+
+// Copyright Programmfabrik GmbH
+// All Rights Reserved
+
+// The gotmplx command wires up variables into a go template and renders it as output
+
+import (
+	"fmt"
+	"html/template"
+	"io/ioutil"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/Masterminds/sprig"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	rootCmd = &cobra.Command{
+		Use:     "gotmplx",
+		Short:   "gotmplx: Command line tool to render a go template",
+		Args:    cobra.MaximumNArgs(1),
+		Version: "1.0",
+		Run:     render,
+		PreRun:  parseVariables,
+	}
+	vars                 []string
+	csvs                 []string
+	eval                 string
+	templateEnvVariables map[string]interface{}
+	templateVariables    map[string]interface{}
+	templateCSVVariables map[string][]map[string]interface{}
+)
+
+func init() {
+	rootCmd.Flags().StringArrayVarP(&vars, "var", "", []string{}, "Parse and use variable in template")
+	rootCmd.Flags().StringArrayVarP(&csvs, "csv", "", []string{}, "Parse and use CSV file rows in template")
+	rootCmd.Flags().StringVarP(&eval, "eval", "e", "", "Parse this text instead of file argument")
+}
+
+func main() {
+	err := rootCmd.Execute()
+	if err != nil {
+		fmt.Fprintln(rootCmd.OutOrStderr(), err)
+		os.Exit(1)
+	}
+}
+
+func parseVariables(cmd *cobra.Command, args []string) {
+
+	envStr := os.Environ()
+	templateEnvVariables = make(map[string]interface{})
+	for _, v := range envStr {
+		parts := strings.Split(v, "=")
```

Comment:
This breaks here: 

```
./gotmplx --var a=b --var c=d=9 -e '{{ .Var.c }}'
```

Output "d". Expected "d=9"

Add a unit test for this.